saleor/plugins/invoicing/plugin.py

```diff
@@ -17,6 +17,14 @@ class InvoicingPlugin(BasePlugin):
     PLUGIN_DESCRIPTION = "Built-in saleor plugin that handles invoice creation."
     CONFIGURATION_PER_CHANNEL = False
 
+    @staticmethod
+    def sanitize_invoice_number(invoice_number):
+        """Invoice number contains slashes so it's unsafe as saved file would mimic a path.
+
+        Slashes should be replaced with hyphens.
+        """
+        return invoice_number.replace("/", "-")
```

Comment:
It looks insufficient when it comes to reserved and control characters. Seems like the right place to check: https://docs.aws.amazon.com/AmazonS3/latest/userguide/object-keys.html

Reply:
As noted in Slack, please use slugify instead.